### DISCUSSION TITLE
Sidepanel-WebUI (like "Google-Backup") planned?

### DISCUSSION THREAD

**Discussion by SmilyHM** [lavinir/hassio-onedrive-backup#39] [Ideas]
Hello everyone,

1st off all: Thank you for this AddOn! :-)

I'm in the process of switching from "Home Assistant Google Drive Backup" (https://github.com/sabeechen/hassio-google-drive-backup) to "OneDrive Backup". With this in mind, I am of course very spoiled by the WebUI and management of Google Drive Backup. 

It would be great if the OneDrive-AddOn offered at least a very similar WebUI (Sidepanel) in the future - including management of backups and immediate start of a backup (#36), and so on.

 If not, then of course it's not the end of the world - it's just a feature request :)

Best regards
Sascha

**Comment by lavinir** [comment]
Thanks for the feedback. There is a plan to add a WebUI to the addon but cannot commit to an ETA currently :)

**Comment by lavinir** [comment]
@SmilyHM  -a WebUI has been added to the latest release (2.0)
